src/gmail/interfaces.ts

```diff
@@ -1,18 +1,18 @@
-import { ILayout } from "../shared/interfaces"
+import { ILayout } from "../shared/interfaces";
 
 export interface IItemGmail {
-  check:boolean,
-  star:boolean
+  check: boolean,
+  star: boolean,
 }
 
 export interface IGmail extends ILayout {
-  drafts:Array<IItemGmail>,
-  inbox:Array<IItemGmail>,
-  sent:Array<IItemGmail>,
-  important:Array<IItemGmail>,
-  snoozed:Array<IItemGmail>,
-  spam:Array<IItemGmail>,
-  emails:Array<IItemGmail>,
-  check:boolean,
-  url:string
-}
\ No newline at end of file
+  drafts: IItemGmail[],
```

Comment:
I find `Array<>` more explicit, since this was already used, perhaps keep this rule?